happi/tests/test_cli.py

```diff
@@ -79,6 +83,14 @@ def test_cli_version(capsys):
     assert happi.__file__ in readout.out
 
 
+def test_cli_no_argument(capsys):
+    happi.cli.happi_cli([])
+    readout = capsys.readouterr()
+    assert 'usage:' in readout.out
+    assert '[-h] [--path PATH] [--verbose] [--version]' in readout.out
+    assert '{search,add,edit,load}' in readout.out
```

Comment:
I think very specific reflexive tests like this are a maintenance hassle. What if argparse changes its output format slightly? What if we add an argument?

In cases like this I think less is more, e.g. just asserting that `usage` is in there would have been okay. If there's a need to make it specific, we should instead be getting the parser and calling `parser.print_usage()`, to make sure it matches what capsys sees. That way at least we don't have to modify this test for every cli interface update.

Reply:
Ok that sounds good, i'll update to only check for `usage` for now...